composer.json

```diff
@@ -14,6 +14,7 @@
     "minimum-stability": "dev",
     "require": {
         "php": ">=5.3.9",
+        "doctrine/orm": "~2.2,>=2.2.3,<2.5",
```

Comment:
this was taken straight from symfony standard edition 2.7:
https://github.com/symfony/symfony-standard/blob/2.7/composer.json#L12


but indeed, we should simply say `^2.2.3` and leave it to the
application to know if they have an issue with the orm or not.